### PR TITLE
Update .gitignore.

### PR DESCRIPTION
**Pull request by whirm**
Don't ignore files that the tests aren't generating anymore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,11 +12,6 @@
 /.project
 /.pydevproject
 /TAGS
-/Tribler/Test/data/*.sdb
-/Tribler/Test/test_.Tribler/
-/Tribler/Test/test_.Tribler2
-/Tribler/Test/test_TriblerDownloads2
-/Tribler/community/tunnel/.Tribler*
```
Comment (rjruigrok) [inline .gitignore]: `-/Tribler/community/tunnel/.Tribler*` is still being used, by community/tunnel/main.py

Comment (whirm) [inline .gitignore]: You shouldn't keep state data in the code directories. Move it to the home dir or make it a parameter.

Comment (rjruigrok) [inline .gitignore]: It has always been there. :) 
I'll fix this in my next PR.

Comment (whirm) [inline .gitignore]: Glad it's not anymore :D

**Comment by whirm** [comment]
@LipuFei ready

**Pull request merged**